tests/test_ratlib.py

```diff
@@ -24,7 +24,7 @@
      'Um die Rat zu deiner Freundesliste hinzuzufügen drücke'),
     ('которые отдают топливо, нажмите здесь: http://t.fuelr.at/kgbfoamru',
      'которые отдают топливо, нажмите здесь: http://t.fuelr.at/kgbfoamru'),
-    ('a;A;B;b;C;c;1\'2\'3', 'aABbCc123'),
+    ('a;A;B;b;C;c;123', 'aABbCc123'),
```

Comment:
??? what does this fix?

Reply:
No further reason to strip quotes or semi-colons, as we're not vulnerable to this type of injection due to composed strings.